spec/daru/io/importers/redis_spec.rb

```diff
@@ -0,0 +1,66 @@
+RSpec.describe Daru::IO::Importers::Redis do
+	let(:connection) { Redis.new {} }
+	subject { Daru::IO::Importers::Redis.new(connection, *keys).load }
+
+  after { connection.flushdb }
```

Comment:
The current version of `daru` has completely excluded it's specs from rubocop's vision. I copied the same config (rookie mistake), and just now ran rubocop on this spec file to look at 370+ rubocop offenses. I'm fixing them now. 😢